go.mod

```diff
@@ -58,8 +58,8 @@ require (
 replace (
 	github.com/petermattis/goid => github.com/petermattis/goid v0.0.0-20240327183114-c42a807a84ba
 	golang.org/x/crypto => golang.org/x/crypto v0.19.0
```

Comment:
Should push crypto to 0.21 and update to the new SI when that change is committed.